vllm/engine/metrics.py

```diff
@@ -185,11 +191,16 @@ class Stats:
     # Request stats (should have _requests suffix)
     #   Latency
     time_e2e_requests: List[float]
+    time_queue_requests: List[float]
```

Comment:
looks like none of these are actually being logged to prometheus?

Reply:
yes. sorry. I forget. Have modified